ionic_user_interface/src/common/api/route/getRoutesByUser.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { routeBaseUrl } from './config';
```

Comment:
This and the config file seems a little unnecessary lol

Reply:
This was in case we hav many microservices then in the end decide to map them all to domain for example `routemaker.rocks/api`. Yea I ll just remove it since we are not doing that